docs/monitoring/metrics.md

```diff
@@ -1139,6 +1139,12 @@ Metrics related to data exchange between task executors using netty network comm
       <td>An estimate of the output buffers usage.</td>
       <td>Gauge</td>
     </tr>
+    <tr>
+      <td rowspan="1">Shuffle.Netty.BackPressure</td>
```

Comment:
I think this should be moved out from `Netty` scope. This metric will work for all of the shuffle services, not only the netty so:

- could you create a separate table for the `isBackPressured` metric, titled `Common shuffle service metrics`, with a single metric inside `Shuffle.BackPressure.isBackPressured`?
- adjust the code accordingly?

Reply:
Ok, i'll update it.

Reply:
It shouldn't be `Shuffle.Netty.Backpressure`, because `Shuffle.Netty` contains netty specific metrics. Since `Task#isBackPressured` method is using `ResultPartitionWriter#getAvailableFuture`, which is part of the shuffle API, `Task#isBackPressured`  will work automatically for every shuffle service, not only netty.

By the way, I'm not sure if I was clear enough about moving this to a separate table. I meant currently you added `isBackPressured` metric to the:
https://ci.apache.org/projects/flink/flink-docs-stable/monitoring/metrics.html#default-shuffle-service
> Default shuffle service

section, while because of the above reasoning, it should be in a new section and a new table (currently with only single entry) titled:
> Common shuffle service metrics

Reply:
Ok, get it.